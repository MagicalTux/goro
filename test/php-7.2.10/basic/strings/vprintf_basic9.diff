     *** Testing vprintf() : basic functionality - using hexadecimal format ***
     b
     int(1)
004- B
005- int(1)
004+ %X
005+ int(2)
     b 84
     int(4)
008- B 84
009- int(4)
008+  
009+ int(1)
     b 84 b1
     int(7)
012- B 84 B1
013- int(7)
012+   
013+ int(2)
     ===DONE===
