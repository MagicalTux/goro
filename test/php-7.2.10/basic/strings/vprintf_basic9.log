
---- EXPECTED OUTPUT
*** Testing vprintf() : basic functionality - using hexadecimal format ***
b
int(1)
B
int(1)
b 84
int(4)
B 84
int(4)
b 84 b1
int(7)
B 84 B1
int(7)
===DONE===
---- ACTUAL OUTPUT
*** Testing vprintf() : basic functionality - using hexadecimal format ***
b
int(1)
%X
int(2)
b 84
int(4)
 
int(1)
b 84 b1
int(7)
  
int(2)
===DONE===
---- FAILED
