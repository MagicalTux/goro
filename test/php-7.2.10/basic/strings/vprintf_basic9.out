*** Testing vprintf() : basic functionality - using hexadecimal format ***
b
int(1)
%X
int(2)
b 84
int(4)
 
int(1)
b 84 b1
int(7)
  
int(2)
===DONE===